default_metadata/component/checkboxes.json

```
@@ -0,0 +1,9 @@
+{
+  "_id": "definition.checkboxes",
+  "_type": "checkbox",
+  "errors": {},
+  "hint": "Component hint",
```

```suggestion
  "hint": "[Hint text (optional)]",
```

double check the prototype, but i think that's the text fabien wanted